server/src/main/java/org/opensearch/OpenSearchException.java

```diff
@@ -655,7 +655,7 @@ public static OpenSearchException[] guessRootCauses(Throwable t) {
              * parsing exception because that is generally the most interesting
              * exception to return to the user. If that exception is caused by
              * an OpenSearchException we'd like to keep unwrapping because
-             * ElasticserachExceptions tend to contain useful information for
+             * OpenSearchException tend to contain useful information for
```

Comment:
Should this be `OpenSearchException's`? We're talking about multiple here. Or `OpenSearchException tends`?

Reply:
Should be plural, yes